### ISSUE TITLE
[Feature] Skip link, no skip link fails WCAG 2.4.1

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#111]
Level A 2.4.1 - A mechanism is available to bypass blocks of content that are repeated on multiple Web pages.
There should be a method for keyboard only users to skip content that is repeated on multiple Web pages.
There should be a skip link at the top of the page to the main content of the page as the header is repeated across all the sites pages.

**Comment by ecstaticrainbow** [comment]
I will take this to the UX team and get something designed up 
TBH, it could look something like this just admiralty blue
<img width="385" alt="image" src="https://github.com/UKHO/admiralty-design-system/assets/24593464/f86882bf-46e9-4bf5-840c-b68a95ab3456">

**Comment by ecstaticrainbow** [comment]
this is our current admiralty.co.uk site. Any guidance on this would be appreciated with which design is better
<img width="954" alt="image" src="https://github.com/UKHO/admiralty-design-system/assets/24593464/5e79dbd7-bda8-42c6-906a-61fd8b35e343">
